### DISCUSSION TITLE
Transfers API

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#4039] [General]
Does this new API allow a seller to request a payout transfer at any time, given there are available funds in the payout account? Or, does the two week payout window still apply to the seller? The API documentation does not specify.

**Comment by mafge** [comment]
Hi @kevnhowe,
thanks for reaching out!
Transfer API allows you to request a payout on demand and is not bound to the usual two week cycle. Please be aware that you can only trigger a payout once every 24 hours and your account allows a payout (valid deposit method, outstanding balance higher than minimum permissible payout amount etc.).
Best,
Marc
Selling Partner Developer Services
